tests/test_authentication.py

```
@@ -86,54 +95,81 @@ def close(self):
 
     def test_login_failure_lockout(self):
         # Simulate multiple failed login attempts to trigger lockout
-        login_attempts = {} # reset 
+        login_attempts = {}  # reset
```

Ruff (0.11.9)

Redefinition of unused `login_attempts` from line 16

(F811)